### ISSUE TITLE
Options screen jumps to top every few seconds

### ISSUE THREAD

**Issue by ISibboI** [Aktanusa/CookieMonster#997]
### Bug description

When trying to look through the options, the scroll area jumps to the top every few seconds. This makes it impossible to see or set any of them.

### To reproduce

1. Open cookie clicker
2. add cookie monster via javascript bookmark
3. click options
4. scroll down
5. wait a few seconds
 -> options jump back to top

### Save file

[SibboBakery.txt](https://github.com/CookieMonsterTeam/CookieMonster/files/7187173/SibboBakery.txt)

### Browser / Steam / Version

Chrome Version 93.0.4577.82 (official build) (64-Bit) / Firefox 92.0 (64-Bit)
Cookie Clicker 2.031
Cookie Monster 2.031.10

**Comment by dragonrider7225** [comment]
In my experience, it doesn't jump to the top of the options menu, it jumps to `current_options_height.clamp(0, vanilla_options_height)`.

**Comment by Aaeeschylus** [comment]
This happens in the steam version too.

**Comment by DanielNoord** [comment]
Duplicate of #974 and #961.
I'm waiting on the future update to the Modding API as discussed by Orteil. I hope it includes a hook to provide option menu sections.

**Comment by DanielNoord** [comment]
This should now be fixed! Let me know if there are any remaining problems!
